transcom/mymove
# [MB-13917] BUG: Redirect TIOs back to the payment request page from view orders and allowances view

## [Jira ticket](https://dp3.atlassian.net/browse/MB-13917)

## Summary

This PR makes it so that if a TIO closes out of the orders pane or view allowance pane they are redirected back to the previous page - the payment request page and not the move details page.

At the same time, if an office user is on the move details page and clicks Edit Orders, if the X out of the pane (while viewing either orders or allowances) they should be taken back to the previous page (move details)

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the office app
2. Login as a TIO
3. Select a payment request that has a Pending Status (these show up in the queue as "Payment Requested"
4. On the payment request tab, scroll to find the "View Orders" link for the payment request and click it
5. Ensure that clicking X on View Orders takes you back to the previous page (payment request tab). 
6. Click on View Orders again and the click on View Allowances. Ensure that clicking X takes you back to the payment request page
7. Navigate to the move details page and click Edit Orders
8. Ensure that clicking X takes you back to the previous page (move details tab)
9. Click on Edit Orders again and click on View Allowances. Ensure that clicking X takes you back to the move details page

### Frontend


## Screenshots
Screenshot included for a visual on where in the app to check
<img width="1792" alt="Screenshot 2023-07-19 at 12 03 54 PM" src="https://github.com/transcom/mymove/assets/67110378/436c0e18-25f0-40f7-bbd4-0c6f1ba76bc2">
